generators/app/templates/src/app/components/Routes/index.ejs

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
+import React, { Suspense, lazy } from 'react';
 import { ConnectedRouter } from 'connected-react-router';
 import { Switch } from 'react-router-dom';
 
 import { history } from '../../../redux/store';
-import Home from '../../screens/Dashboard';
+import Loading from '../Spinner/components/loading';
 
 import AuthenticatedRoute from './components/AuthenticatedRoute';
 import * as Routes from './constants';
 import styles from './styles.scss';
+
+const Home = lazy(() => import('../../screens/Dashboard'));
+
```

Comment:
Delete this space because the comment below add a extra space when is bootstrapped